src/drive/mobile/containers/onboarding/Analytics.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import { connect } from 'react-redux'
-import OnBoarding from '../../components/OnBoarding'
```

Comment:
We had 2 components called `OnBoarding`, one dumb and one container. I renamed the dumb component.